### ISSUE TITLE
style: default values for `name` parameter in Cloud Run samples

### ISSUE THREAD

**Issue by rogerthatdev** [terraform-google-modules/terraform-docs-samples#164]
### TL;DR

Currently, in Cloud Run samples, the `google_cloud_run_service` resource  uses various generic values as the `name`, such as`cloudrun-srv`, `cloud-run-service-name`.  For the sake of consistency, these names should be either meaningful to their intended function ([example](https://github.com/terraform-google-modules/terraform-docs-samples/blob/145717145202025d4865ca9d6213cc5a19300f25/cloud_run_service_interservice/main.tf#L5)) or have a default generic value (something like `my-cloud-run-service`).

### Terraform Resources

[google_cloud_run_service](https://registry.terraform.io/providers/hashicorp/google/latest/docs/resources/cloud_run_service)

**Comment by glasnt** [comment]
Since this issue was logged, the only samples that use generic names appear to be: 

* Tasks (being addressed in #568, incidentally)
* run/traffic_* samples
* run/*_tag_* samples

**Comment by msampathkumar** [comment]
Quick update: I was checking to know if this issues is still valid, and I found few areas to improve. So not closing this request.


```
noauth/main.tf:  name     = "cloudrun-srv"
..
remove_tag/main.tf:  name     = "cloudrun-srv"
secret_manager/main.tf:  name     = "cloudrun-srv-env-var-secret"
..
traffic_gradual_rollout/main.tf:  name     = "cloudrun-srv"
traffic_latest_revision/main.tf:  name     = "cloudrun-srv"
traffic_rollback/main.tf:  name     = "cloudrun-srv"
traffic_split/main.tf:  name     = "cloudrun-srv"
traffic_split_tag/main.tf:  name     = "cloudrun-srv"
```
